dali/dfu/dfurun.cpp

```diff
@@ -614,7 +624,7 @@ class CDFUengine: public CInterface, implements IDFUengine
 
         SecAccessFlags perm = queryDistributedFileDirectory().getFDescPermissions(fd,user,auditflags);
         StringBuffer name;
-        ensureFilePermissions(getFDescName(fd,name),perm,write);
+        ensureFilePermissions(true,nullptr,getFDescName(fd,name),perm,write);
```

Comment:
is both, bool legacyPhysicalPerms and const char * planeName params necessary?
Can it be assume that if planeName == nullptr, then it is legacy ?

Reply:
Yes, if planeName == nullptr, it is legacy. The legacyPhysicalPerms will be removed.